<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWESDA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/icon.ico</normaloff>:/resources/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="treeWidget">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="header">
        <property name="font">
         <font>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Для початку роботи відкрийте файл
або додайте елементи</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="subheader">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">hide:true</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;На випадок, якщо ви забули:&lt;/p&gt;&lt;p&gt;- Редагування елеменів за допомогою подвійного кліку&lt;/p&gt;&lt;p&gt;- Видаляти можна будь-який рівень,&lt;/p&gt;&lt;p&gt;- Аби обрати рівень, який не потрібно видаляти,&lt;/p&gt;&lt;p&gt;необхідно обрати значення &amp;quot;null&amp;quot; на цьому рівні&lt;/p&gt;&lt;p&gt;- Пошук здійснюється по усіх полях,&lt;/p&gt;&lt;p&gt;виводяться усі можливі збіги.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>840</width>
           <height>68</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_3">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
        <property name="placeholderText">
         <string>назва</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
        <property name="placeholderText">
         <string>регіон</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
        <property name="placeholderText">
         <string>поштовий індекс</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Створити</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menunfdnfdnfd">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Робота</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Додати</string>
     </property>
     <addaction name="action_7"/>
     <addaction name="action_8"/>
     <addaction name="action_9"/>
    </widget>
    <addaction name="action_3"/>
    <addaction name="menu_2"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action_15"/>
    <addaction name="action_14"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Функції</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
   </widget>
   <addaction name="menunfdnfdnfd"/>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Завантажити з файлу</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Зберегти у файл</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Пошук</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Вилучити</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Створити новий</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Місто</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Магазин</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Товар</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Запаси магазинів за сумарною вартістю всіх товарів</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Сортувати товар на зростання</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Сортувати товар на спадання</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>Пошук у ціновому діапазоні</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Пошук товару у певних магазинах(містах)</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Про програму</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Оновити вміст дерева</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
